docs/DeveloperGuide.adoc

```diff
@@ -297,50 +298,147 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |clerk|view the number of patients who visited the clinic today. |
+
+|`* * *` |clerk|edit patients' details |
+
+|`* * *` |clerk|register new patients with optional fields |
+
+|`* * *` |clerk|find all patients who have missed their appointments. |
+
+|`* * *` |clerk|add ad-hoc patients to the queue |
+
+|`* * *` |clerk|acknowledge appointments if patients are present for their appointments |
+
+|`* * *` |clerk|tag patient with known allergies. |
+
+|`* * *` |clerk|cancel appointments for patients |
+
+|`* * *` |clerk|search for patients using their name or phone number |
+
+|`* * *` |clerk|look up how many patients are in the queue, on a side panel |
+
+|`* * *` |clerk|look up patient using a reference id.
+
+|`* * *` |clerk|reschedule appointments of patients |
+
+|`* * *` |clerk|search for appointment slots easily so that I can schedule appointments for patients easily. |
+
+|`* * *` |clerk|assign a queue number to each patient in the queue |
+
+|`* * *` |clerk|schedule patient's follow up appointments so that I can void long queues. |
+
+|`* * *` |clerk|use the appointment scheduler so that I can schedule appointments for my patients |
+
+|`* * *` |clerk|save time managing the queue so that I have more time to do my own work |
+
+|`* * *` |clerk|take note of the doctors that are on-shift so that I can effectively direct patients to available doctors. |
+
+|`* * *` |experienced user|create custom aliases to commands |
+
+|`* * *` |receptionist|use the undo and redo feature to remdy any mistakes.|
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
 
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `ClerkPro` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Remove person from queue (UC1)
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
-+
+1.  Patient wants to leave
+2.  User requests to remove patient from the queue
+3.  ClerkPro removes the person from queue
+
 Use case ends.
 
 *Extensions*
 
 [none]
-* 2a. The list is empty.
+* 2a. Person is not in queue
 +
 Use case ends.
 
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. ClerkPro shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Add patient into queue (UC2)
+
+*MSS*
+
+1.  New patient arrives at the clinic
+2.  User wants to add new patient into the queue
+3.  ClerkPro adds the patient into the queue
+
+Use case ends.
+
+*Extensions*
+
+* 2a. Actor inputs invalid format
```

Comment:
It might be good if you use consistent terms throughout to refer the the 'Actor' or 'User'. It's a little confusing to interchange them in a single UC